hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -388,7 +388,9 @@ public void deleteVolume(String volumeName) throws IOException {
   @Override
   public void createBucket(String volumeName, String bucketName)
       throws IOException {
-    createBucket(volumeName, bucketName, BucketArgs.newBuilder().build());
+    // Set acls of current user.
+    createBucket(volumeName, bucketName,
+        BucketArgs.newBuilder().setAcls(getAclList()).build());
```

Comment:
This will cause creator acls added twice into the final list because the same list will be added again when the passin bucketArgs has a non-empty acl list. 
{code}
    List<OzoneAcl> listOfAcls = getAclList();
    //ACLs from BucketArgs
    if(bucketArgs.getAcls() != null) {
      listOfAcls.addAll(bucketArgs.getAcls());
    }
{code}

The list of acls are not merged properly when using List#addAll, which will be fixed in HDDS-1927. 
Let's file a separate JIRA for RpcClient#createBucket issue.

Reply:
Done.